tests/test-commands.c

```diff
@@ -244,7 +245,8 @@ static void test_dump_addrs(void const *test_data)
         int const result =
                 mptcpd_kpm_dump_addrs(pm,
                                       dump_addrs_callback,
-                                      L_UINT_TO_PTR(id));
+                                      L_UINT_TO_PTR(id),
+                                      NULL);
```

Comment:
Rather than passing `NULL`, it would be better to pass a pointer to a real function to verify that the `complete` function is called as expected.

Reply:
Done.